taichi/transforms/ir_printer.cpp

```diff
@@ -208,7 +208,7 @@ class IRPrinter : public IRVisitor {
 
   void visit(ContinueStmt *stmt) override {
     if (stmt->scope) {
-      print("{} continue (as_return={})", stmt->name(), stmt->as_return());
+      print("{} continue (scope={})", stmt->name(), stmt->name());
```

Comment:
Will that be good if we also add this to `FrontendBreakStmt`?

Reply:
Yeah i think so. Let's do it in another PR

Reply:
I just realized you said `FrontendBreakStmt`, maybe more useful to add this for `BreakStmt`?